### PR TITLE
fix: lnd listening URIs example, make it clear they are public, raiden -> connext, rm btc ln node

### PR DESCRIPTION
**Pull request by kilrau**
@cjdelisle

### CODE REVIEW

File: 02-peer-protocol.md [hunk 1]
```diff
@@ -163,17 +163,17 @@ The `NodeStateUpdate` message is used to tell a peer about a change in the node
     `repeated string pairs = 2`
     The sender's list of trading pair symbols, constructed with the base currency first, followed by a  '/' separator and the quote currency (e.g., [“LTC/BTC”, “DAI/BTC”])
 
-    `string raiden_address = 3`
-    The sender's Raiden address
+    `string connext_address = 3`
+    The sender's connext address
```
Comment (ghost) [inline 02-peer-protocol.md]: I think this should be named connext public key. Example: `indra123abc`

File: 02-peer-protocol.md [hunk 2]
```diff
@@ -163,17 +163,17 @@ The `NodeStateUpdate` message is used to tell a peer about a change in the node
...
 
     `map<string, string> lnd_pub_keys = 4`
     The sender's list of LND public keys
 
     `map<string, string> token_identifiers = 5`
-    Mapping between currency symbols to token identifiers such as LND chains or Raiden token contract address (e.g., { BTC: 'bitcoin-testnet', LTC: 'litecoin-testnet' })
+    Mapping between currency symbols to token identifiers such as LND chains or connext token contract address (e.g., { BTC: 'bitcoin-mainnet', LTC: 'litecoin-mainnet' })
```
Comment (ghost) [inline 02-peer-protocol.md]: Perhaps ethereum ERC20 token contract address or zero address `0x0000000000000000000000000000000000000000` for ETH is better here?

File: 04-swap-protocol.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Once an order match is found in the taker’s order book, the swap protocol should be initiated. The current swap protocol assumes that the taker and maker are connected via a payment channel network (e.g. the [Lightning](http://lightning.network/) or [Raiden Network](https://raiden.network/)) with sufficient balance available for the swap. The following is the swap protocol's "happy" flow:
+Once an order match is found in the taker’s order book, the swap protocol should be initiated. The current swap protocol assumes that the taker and maker are connected via a payment channel network (e.g. the [Lightning](http://lightning.network/) or [connext Network](https://connext.network/)) with sufficient balance available for the swap. The following is the swap protocol's "happy" flow:
```
Comment (kilrau) [inline 04-swap-protocol.md:5]: Duplicate: https://github.com/opendexnetwork/opendex/pull/20

File: 02-peer-protocol.md (hunk 2 above)
Comment (kilrau) [reply 02-peer-protocol.md]: Like this? If not, please point me to the code snippet in xud where we do that mapping for ETH-ERC20.

File: 02-peer-protocol.md (hunk 1 above)
Comment (kilrau) [reply 02-peer-protocol.md]: Fixed.

**Review by hatmer — APPROVED** [review]

**Pull request merged**